sops/howto/disaster_recovery.asciidoc

```diff
@@ -0,0 +1,34 @@
+
+# Disaster recovery
+
+Most of the service data is stored in a database, some of it in a secrets manager. All state is managed from the control plane component that's running in APP Interface.
+
+The database must be setup in a way that satisfies the data service continuity requirements. Same applies for the secretes manager.
+
+## Steps to recover the service 
+
+To recover the service after a data loss:
+
+1. use the database service (e.g. AWS RDS) data recovery mechanism,
+2. use the secrets manager (e.g. AWS Secrets Manager) data recovery mechanism and
+3. re-start the control plane component.
+
+NOTE: To perform these step, you need to contact an AppSRE IC to carry it out.
+
+## Detailed instructions
+
+General instructions for managing service components through APP Interface can be found here:
+https://gitlab.cee.redhat.com/service/app-interface#our-how-to-guides
+
+The RHOC service components are configured here:
+https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/managed-connectors/namespaces/managed-connectors-production.yaml
+
+Instructions on recovering an RDS instance from a snapshot are available here: https://gitlab.cee.redhat.com/service/app-interface#restoring-rds-databases-from-backups
+
+Instructions on recovering a secrets from a snapshot are available here: 
+https://docs.aws.amazon.com/secretsmanager/latest/userguide/manage_restore-secret.html
+(Pending SOP)
```

Comment:
Where is this SOP to reside? Do we have an open task to complete it?

Reply:
It needs to reside with the other APP SRE ones. Feng wanted to put it together, but I am not sure it happened.